amd/src/mutations.js

```diff
@@ -1,10 +1,16 @@
 import Ajax from 'core/ajax';
+import Notification from 'core/notification';
+import {get_string as getString} from 'core/str';
```

Comment:
You should use getString function directly which uses a native promise instead of a jquery one

Reply:
This is not possible yet without losing backwards compatibility - will be fixed when dropping support for Moodle 4.1

Reply:
can we add a comment noting that?